README.adoc

```diff
@@ -25,6 +25,8 @@ is set, for example by running `export GO15VENDOREXPERIMENT=1`.
 In Go `1.6` it is enabled by default and in Go `1.7` it is always enabled
 without the ability to turn it off.
 
+If on Windows, the `GOPATH` environment variable is not set by default - see http://www.wadewegner.com/2014/12/easy-go-programming-setup-for-windows/[step 5].
```

Comment:
Ok, this wasn't mentioned so far, so I assumed it happened automatically
elsewhere.

On 2 August 2016 at 17:43, Max Rydahl Andersen <notifications@github.com>
wrote:

> In README.adoc
> <https://github.com/almighty/almighty-core/pull/74#discussion_r73191989>:
>
> > @@ -25,6 +25,8 @@ is set, for example by running `export GO15VENDOREXPERIMENT=1`.
> >  In Go `1.6` it is enabled by default and in Go `1.7` it is always enabled
> >  without the ability to turn it off.
> >
> > +If on Windows, the `GOPATH` environment variable is not set by default - see http://www.wadewegner.com/2014/12/easy-go-programming-setup-for-windows/[step 5].
>
> this is not really dependent on windows. Same goes for any GO project. You
> need to setup a $GOPATH for all.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/almighty/almighty-core/pull/74/files/930aa516f15054de98bae54581659840987c4383#r73191989>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJoQU2ndQU9NHsg9IgiWa0grYGlFS_oks5qb3PHgaJpZM4JazQN>
> .
>